src/test/java/uk/gov/pay/directdebit/payments/services/PaymentServiceTest.java

```diff
@@ -133,6 +132,9 @@ public void paymentSubmittedToProvider_shouldUpdatePaymentAsPending_andRegisterA
     public void createShouldCreatePayment() {
         Mandate mandate = mandateFixture.toEntity();
 
+        when(mockedGovUkPayEventService.storeEventAndUpdateStateForPayment(any(Payment.class), any(GovUkPayEventType.class)))
```

Comment:
Should at least be able to use `eq(GovUkPayEventType.PAYMENT_CREATED)` instead of `any(GovUkPayEventType.class)`

Reply:
fixed